.ddev/commands/web/openmage-install

```diff
@@ -0,0 +1,154 @@
+             ;;
+         *)               # Default case: No more options, so break out of the loop.
+             break
+     esac
+
+     shift
+ done
+
+LOCALXML="${ROOT}/app/etc/local.xml"
+if [ -f "${LOCALXML}" ]; then
+  if [[ "${QUIET_INSTALL_FLAG}" ]]; then
+    DELETE='y'
+  else
+    read -r -p "OpenMage is already installed. Delete local.xml and drop database [y/N] " DELETE
+    DELETE=${DELETE,,} # to lower
+  fi
+
+  if [[ "${DELETE}" =~ ^(yes|y) ]]; then
+    mysql -u db -h db -e "DROP SCHEMA IF EXISTS db; CREATE SCHEMA db;";
+    rm "${LOCALXML}"
+  else
+    exit 1
+  fi
+fi
+
+# install sample data
+if [[ "${SAMPLE_DATA_FLAG}" ]]; then
+  INSTALL_SAMPLE_DATA='y'
+else
+  read -r -p "Install sample data? [y/N] " INSTALL_SAMPLE_DATA
+  INSTALL_SAMPLE_DATA=${INSTALL_SAMPLE_DATA,,} # to lower
+fi
+
+if [[ $INSTALL_SAMPLE_DATA =~ ^(yes|y) ]]; then
+  SAMPLE_DATA_URL=https://github.com/Vinai/compressed-magento-sample-data/raw/master/compressed-magento-sample-data-1.9.2.4.tgz
+  SAMPLE_DATA_DIRECTORY="${ROOT}/.sampleData"
+  SAMPLE_DATA_FILE=sample_data.tgz
+
+  if [[ ! -d "${SAMPLE_DATA_DIRECTORY}" ]]; then
+    echo "Creating backup directory ..."
+    mkdir -p "${SAMPLE_DATA_DIRECTORY}"
+  fi
+
+  cd "${SAMPLE_DATA_DIRECTORY}" || exit
+  if [[ ! -f "${SAMPLE_DATA_DIRECTORY}/${SAMPLE_DATA_FILE}" ]]; then
+    echo "Downloading sample data ..."
+    wget "${SAMPLE_DATA_URL}" -O "${SAMPLE_DATA_FILE}"
+  fi
+
+  echo "Uncompressing sample data ..."
+  tar xf "${SAMPLE_DATA_FILE}"
+
+  echo "Copying sample data ..."
+  cp -r magento-sample-data-1.9.2.4/* "${PWD}/"
+
+  echo "Importing sample data ..."
+  mysql -u db -h db db < "${SAMPLE_DATA_DIRECTORY}/magento-sample-data-1.9.2.4/magento_sample_data_for_1.9.2.4.sql"
+
+  # remove sample data
+  if [[ "${SAMPLE_DATA_KEEP_FLAG}" ]]; then
+    rm -rf $(
+      find . -maxdepth 1 -type f -name "*" ! -name "${SAMPLE_DATA_FILE}")
+  else
+    cd "${ROOT}" || exit
+    rm -rf "${SAMPLE_DATA_DIRECTORY}"
+  fi
+fi
+
+cd "${ROOT}" || exit
+
+if [[ "${USE_DEFAULT_FLAG}" ]]; then
+  ADMIN_USER='admin'
+  ADMIN_FIRSTNAME='OpenMage'
+  ADMIN_LASTNAME='Administrator'
+  ADMIN_EMAIL='admin@example.com'
+  ADMIN_PASSWORD='veryl0ngpassw0rd'
+  TABLE_PREFIX='om_'
+else
+  read -r -p "Admin user [admin]: " ADMIN_USER
+  ADMIN_USER=${ADMIN_USER:-admin}
+  read -r -p "Admin firstname [OpenMage]: " ADMIN_FIRSTNAME
+  ADMIN_FIRSTNAME=${ADMIN_FIRSTNAME:-OpenMage}
+  read -r -p "Admin lastname [Administrator]: " ADMIN_LASTNAME
+  ADMIN_LASTNAME=${ADMIN_LASTNAME:-Administrator}
+  read -r -p "Admin email [admin@example.com]: " ADMIN_EMAIL
+  ADMIN_EMAIL=${ADMIN_EMAIL:-admin@example.com}
+  read -r -p "Admin password [veryl0ngpassw0rd]: " ADMIN_PASSWORD
+  ADMIN_PASSWORD=${ADMIN_PASSWORD:-veryl0ngpassw0rd}
+  if [[ $INSTALL_SAMPLE_DATA =~ ^(yes|y) ]]; then
+    TABLE_PREFIX=''
+  else
+    read -r -e -i 'om_' -p "Table prefix [om_]: " TABLE_PREFIX
```

Comment:
I do not use a prefix in the name of the tables. If I press Enter it will add om_ to all table and I don't want that. It is better to leave the previous version (empty) and whoever wants a prefix to enter it.

Reply:
I case of testing 3rd-party code its really useful to test with prefix set. I'd like to keep it.

Reply:
I would like this script not to condition me in the choices I make.

Reply:
This script is only for testing. I want to have it set by default to find error like this ... https://magento.stackexchange.com/a/167944/46249

I'll keep it.

Reply:
I consider this script very useful and I understand that you only want to test the version with tables having a prefix. That's why I have nothing against it. But the file system and database can be used later in production with a server configuration other than DDEV. That's what I am doing at the moment, with a shop that I have to get going in the next period. I did not use a prefix for the tables. 

If I were to test your installation script without Sample Data, I would not be able to change the prefix except by editing the file with sed. For this reason, I would prefer to be able to install without a prefix. You could mention that you recommend using a table prefix to cover this situation as well.